single-audit/resolve_findings/templates/resolve_findings/finding-resolution-page.html

```diff
@@ -2,20 +2,7 @@
 {% load static %}
 
 {% block content %}
-  <div id="app_header" class="grid-row padding-y-5">
-    <div class="grid-col-1"></div>
-    <div class="grid-col-8">
-      <h1 id="app_title" class="font-serif-lg text-italic">Single Audit Resolution Solution</h1>
-    </div>
-    <div id="logged_in_username" class="grid-col-2">
-      <span class="text-middle">Tom Hammersmith</span>
-      <img alt="user icon"
-           src="{% static 'img/user-circle-gray.svg' %}"
-           height="24"
```

Comment:
Is specifying a static number (`24`) kosher here?

Reply:
@fureigh Hm, I don't see why not. It's a small enough icon, I don't think it needs to be sized differently at different screen widths. Open to suggestions for how to size this more elegantly or idiomatically though — I want to learn!